protocol/messenger_mailserver.go

```diff
@@ -669,6 +722,31 @@ func (m *Messenger) LoadFilters(filters []*transport.Filter) ([]*transport.Filte
 	return m.transport.LoadFilters(filters)
 }
 
+func (m *Messenger) ToggleUseMailservers(value bool) error {
+	err := m.settings.SetUseMailservers(value)
+	if err != nil {
+		return err
+	}
+
+	if value {
+		m.cycleMailservers()
+		return nil
+	}
```

Comment:
In a future PR, we could call instead `m.StartMailserverCycle()` and also implement a `m.StopMailserverCycle()` which should stop these goroutines:
```
go m.checkMailserverConnection()
go m.updateWakuV1PeerStatus()
go m.updateWakuV2PeerStatus()
```
That way we ensure that the mailserver cycle is not really running.